runtimevar/blobvar/blobvar.go

```diff
@@ -16,6 +16,19 @@
 // variables read from a blob.Bucket.
 // Use NewVariable to construct a *runtimevar.Variable.
 //
+// URLs
+//
+// For runtimevar.OpenVariable URLs, blobvar registers for the scheme "blob".
+// The host+path is used as the blob key. blobvar supports the following URL
+// parameters:
+//   - bucket: The URL to be passed to blob.OpenBucket (required).
```

Comment:
I would expect the reverse of this: I would expect the key to be in a query parameter. It would be nice if this was: `blob+file:///foo/bar&key=myvar.txt&decoder=string`

(I realize this would mean that this would mean the full scheme might be `runtimevar+blob+file`, but I think I'm okay with that.)

Reply:
I agree that would be cool. It's tricky though, because while `blob+file` is certainly one option, any `blob` provider can be used, so `blob+s3` should work as well. `blobvar` doesn't know which set of `blob` providers are registered.

We could try to fix that by having `blob` expose the set of schemes that are registered...? But even then, it will be tricky to separate the URL parameters that are for the `OpenBucket` URL from the ones that are for the `blobvar` URL.

Otherwise, given that the scheme here is `blob`, I think that the only way to encode the URL to be passed to `blob` is in a URL parameter.

Reply:
The more I think about this, the more I think I would like to get it to work. Escaping URLs in URL parameters is icky, and there are several other providers where URLs are needed (e.g., `etcdvar` for the etcd server), and it would be nice to be able to use the main URL itself more naturally (e.g., `etcd+http://my.etcd.server:8080?var=myvar` instead of the current `etcd://myvar?client=http://my.etcd.server:8080`). I filed #1488 to follow up.

Reply:
SGTM.